lib/vm/opFns.js

```diff
@@ -576,7 +617,8 @@ module.exports = {
       trap(ERROR.STATIC_STATE_CHANGE)
     }
 
-    var data = memLoad(runState, inOffset, inLength)
+    subMemUsage(runState, inOffset, inLength)
+    const data = runState.memory.read(inOffset.toNumber(), inLength.toNumber())
```

Comment:
Why are these zero checks not necessary here?

Reply:
Good question, because no test failed for them :sweat_smile:

Fixed all reads that had no length check, thanks for catching!

Reply:
😄